apis/v1alpha1/gatewayclass_types.go

```diff
@@ -85,10 +85,21 @@ type GatewayClassSpec struct {
 type GatewayClassConditionType string
 
 const (
-	// GatewayClassConditionStatusInvalidParameters indicates the
-	// validity of the Parameters set for a given controller. This
-	// will initially start off as "Unknown".
-	GatewayClassConditionStatusInvalidParameters GatewayClassConditionType = "InvalidParameters"
+	// GatewayClassConditionStatusAccepted indicates that the GatewayClass
+	// has been accepted by the controller set in the `spec.controller`
+	// field.
+	//
+	// It defaults to Unknown, and MUST be set by a controller when it sees
+	// a GatewayClass using its controller string.
+	// It must be set to `status: true` if the controller is using the class,
+	// and `status:false` if the controller is not using the class for some reason.
```

Comment:
This feels a bit ambiguous. I think some might interpret "using the class" as "the controller is using the class to render at least one Gateway". Put another way, if no Gateway is trying to use this class, is the class actually used? If I'm interpreting this correctly, I think something like this might be a bit clearer:

> The status of this condition MUST be set to true if the controller will support provisioning Gateways using this class. Otherwise, this status MUST be set to false.